packages/react-components/react-motions-preview/stories/CreateMotionComponent/CreateMotionComponentDefault.stories.tsx

```diff
@@ -16,6 +16,7 @@ const useClasses = makeStyles({
   },
   item: {
     backgroundColor: tokens.colorBrandBackground,
+    ...shorthands.border('3px', 'solid', tokens.colorTransparentStroke),
```

Comment:
nit: why not a token for the width as well?

```suggestion
    ...shorthands.border(tokens.strokeWidthThicker, 'solid', tokens.colorTransparentStroke),
```

Reply:
Done! I ended up cleaning up some other 3px border-width defs in the same package's stories too